fs-repo-11-to-12/migration/swapper.go

```diff
@@ -19,6 +21,32 @@ var SyncSize uint64 = 20 * 1024 * 1024 // 20MiB
 // migration.
 var NWorkers int = 4
 
+func init() {
+	workerEnvVar := "IPFS_FS_MIGRATION_11_TO_12_NWORKERS"
+	syncSizeEnvVar := "IPFS_FS_MIGRATION_11_TO_12_SYNC_SIZE"
```

Comment:
Figured long names would prevent conflict and almost nobody should need to set these, but if you have other names in mind that's fine by me.